src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -26,29 +23,23 @@
   private RelativeEncoder KShooterEncoder;
   private RelativeEncoder KHoodEncoder;
   private MotorController cargo_motorController;
-  private SparkMaxPIDController kCargoController;
-  private RelativeEncoder kCargoEncoder;
-  private NetworkTableEntry sbShooterRPM;
   private double currentRPM;
 
   public ShooterSubsystem() {
-    sbShooterRPM = RobotContainer.debugTab.add("shooterRPM", 0).getEntry();
-
+    SmartDashboard.putNumber("RPMChange", 3000.0);
```

Comment:
Think this was removed, and not committed?  Didn't work, maybe?

Reply:
It was a test attempt by someone that never worked but was never removed, I think